go/mysql/mysql.go

```diff
@@ -38,6 +38,87 @@ func init() {
 }
 
 const (
+	// TypeTiny specifies a TINYINT type.
```

Comment:
I'm wondering if these should be defined in the proto3 file only, so they're visible from all languages... I know you wouldn't get access to the C. stuff then, but we'll need these in all client languages anyway, right?